### PR TITLE
Imports, Lint Fixes and Resolve test cases after new imports

### PR DESCRIPTION
**Pull request by vikas-cis**

### CODE REVIEW

File: src/app/modals/coldstake/coldstake.component.spec.ts [hunk 1]
```diff
@@ -2,12 +2,10 @@ import { async, ComponentFixture, TestBed } from '@angular/core/testing';
...
 import { MdDialogRef, MdSnackBarModule } from '@angular/material';
 
-import { SharedModule } from '../../shared/shared.module'; // fix
-import { RpcModule } from '../../core/rpc/rpc.module';
 import { ModalsModule } from '../modals.module';
-
-import { SnackbarService } from '../../core/snackbar/snackbar.service';
-import { IpcService } from '../../../core/ipc/ipc.service';
+import { CoreUiModule } from '../../core-ui/core-ui.module';
+import { CoreModule, SnackbarService, IpcService } from '../../core/core.module';
```
Comment (kewde) [inline src/app/modals/coldstake/coldstake.component.spec.ts]: You can remove IpcService & snackbar service, singletons provided by CoreModule

File: src/app/modals/coldstake/coldstake.component.spec.ts [hunk 2]
```diff
@@ -21,16 +19,15 @@ describe('ColdstakeComponent', () => {
...
+        CoreModule.forRoot(),
         ModalsModule,
         MdSnackBarModule
       ],
       providers: [
-        { provide: MdDialogRef},
-
-        SnackbarService
+        { provide: MdDialogRef },
+        SnackbarService,
```
Comment (kewde) [inline src/app/modals/coldstake/coldstake.component.spec.ts]: CoreModule.forRoot() takes care of this

File: src/app/core-ui/main/status/status.component.spec.ts [hunk 3]
```diff
@@ -1,23 +1,25 @@
...
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ StatusComponent ],
       imports: [
         SharedModule,
         CoreModule.forRoot(),
         ModalsModule,
+        CoreUiModule.forRoot(),
         MdTooltipModule
```
Comment (kewde) [inline src/app/core-ui/main/status/status.component.spec.ts]: tooltip already in CoreUi

File: src/app/modals/createwallet/createwallet.component.spec.ts [hunk 4]
```diff
@@ -20,6 +22,7 @@ describe('CreateWalletComponent', () => {
         SharedModule,
         CoreModule.forRoot(),
         ModalsModule,
+        CoreUiModule.forRoot(),
         MdSnackBarModule
```
Comment (kewde) [inline src/app/modals/createwallet/createwallet.component.spec.ts]: snackbar in CoreUi

File: src/app/modals/createwallet/passphrase/passphrase.service.spec.ts [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
+import { ModalsModule, PassphraseService } from '../../modals.module';
 
 describe('PassphraseService', () => {
   beforeEach(() => {
     TestBed.configureTestingModule({
       imports: [
         SharedModule,
-        RpcModule.forRoot()
+        CoreModule.forRoot(),
+        ModalsModule.forRoot()
```
Comment (kewde) [inline src/app/modals/createwallet/passphrase/passphrase.service.spec.ts]: ` src/app/modals/createwallet/passphrase/passphrase.service.spec.ts`

Might want to steer clear of importing ModalsModule in the test of a subcomponent of it.

File: src/app/modals/createwallet/passphrase/passphrase.service.spec.ts [hunk 6]
```diff
@@ -1,17 +1,17 @@
...
       ],
       providers: [PassphraseService, IpcService]
```
Comment (kewde) [inline src/app/modals/createwallet/passphrase/passphrase.service.spec.ts]: no need for ipc

File: src/app/core-ui/main/status/status.component.spec.ts (hunk 3 above)
Comment (vikas-cis) [reply src/app/core-ui/main/status/status.component.spec.ts]: yes I need to remove all angular material stuffs from the specs as I already included CoreUiModule, the reason why I remain it due to other test cases which are reflected due to new imports.

**Pull request merged**